package.json

```diff
@@ -31,7 +31,7 @@
 		"eslint-plugin-react": "^7.35.0",
 		"husky": "^9.1.4",
 		"jsdom": "^24.1.1",
-		"lint-staged": "^15.2.8",
+		"lint-staged": "^15.2.9",
```

Comment:
I think this is fine to leave in this PR, but generally speaking its best to try to avoid upgrades to dependencies unless they're strictly related to the specific task at hand 😄